wkd/kustomization.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: wkd
+resources:
+  - github.com/drGrove/docker-wkd/kustomize/direct?ref=7d81b14e809401b8b163b3278e41254b1a05b2be
+  - ingress.yaml
+patchesJson6902:
+  - target:
+      group: apps
+      version: v1
+      kind: Deployment
+      name: wkd
+    patch: |-
```

Comment:
why use a 6092 patch instead of a strategic merge patch?

Reply:
Patch Strategic Merge does not work with adding to an array, unless you override the entire array